README.adoc

```diff
@@ -73,19 +73,28 @@ There are several configuration options that the asciidoctor-maven-plugin uses,
 sourceDirectory:: defaults to `${basedir}/src/main/asciidoc`
 sourceDocumentName:: an override to process a single source file; defaults to all files in `${sourceDirectory}`
 sourceDocumentExtensions:: (named `extensions` in v1.5.3 and below) a `List<String>` of non-standard file extensions to render. Currently ad, adoc, and asciidoc will be rendered by default
-resources:: list of resource files to copy to the output directory (e.g., images, css). By default everything is copied
+resources:: list of resource files to copy to the output directory (e.g., images, css). The configuration follows the same patterns as the `maven-resources-plugin`
 +
 [source, xml]
 ----
     <resources>
         <resource>
+            <!-- (Mandatory) Directory to copy from. By default uses maven's ${baseDir} -->
             <directory>DIRECTORY</directory>
-            <include>**/*.jpg</include>
-            <include>**/*.gif</include>
-            <exclude>**/.svn</exclude>
+            <!-- (Optional) Directory to copy to. By default uses the option `sourceDirectory` -->
```

Comment:
I don't think this comment is correct. The resources aren't copied to the source directory but rather the output directory, correct?